hyp3_sdk/hyp3.py

```diff
@@ -37,12 +54,25 @@ def get_jobs(self, start: datetime = None, end: datetime = None, status: str = N
         return self.session.get(urljoin(self.url, '/jobs'), params=params).json()
 
     def submit_jobs(self, jobs: List[Job], validate_only: bool = False) -> dict:
```

Comment:
`Optional[bool]`?

Reply:
Not optional here as in it shouldn't be `None` -- defaults to False